### DISCUSSION TITLE
red warning text on build

### DISCUSSION THREAD

**Discussion by quetzalsly** [arthelokyo/astrowind#475] [General]
```
00:50:51 [build] Building static entrypoints...
(node:11544) Warning: To load an ES module, set "type": "module" in the package.json or use the .mjs extension.
(Use `node --trace-warnings ...` to show where the warning was created)
(node:11544) Warning: To load an ES module, set "type": "module" in the package.json or use the .mjs extension.
00:50:54 [astro-icon] Loaded icons from flat-color-icons, tabler
```

Is this supposed to happen during build? it wants .mjs somewhere?
I am not sure how to debug this.
I also have "type": "module" in the package.json but it still gives the warning.

**Comment by prototypa** [comment]
Hi @quetzalsly 

Please, more details. No warning is displayed in local builds and _vercel_ builds. Maybe it's something you've modified or it depends on your environment.

Give more details so we can help you.

In the meantime let's move this to discussion.
